main.c

```diff
@@ -57,7 +57,7 @@ void print_help(const char *program) {
     puts("  -m, --mountdir=MOUNTDIR    the mountdir of OverlayFS (optional)");
     puts("  -L, --lowernew=LOWERNEW    the lowerdir of new OverlayFS (optional)");
     puts("  -U, --uppernew=UPPERNEW    the upperdir of new OverlayFS (optional)");
-    puts("  -i  --ignore-mounted       don't prompt if OverlayFS is still mounted (optional)");
+    puts("  -i  --ignore               don't prompt if OverlayFS is still mounted (optional)");
```

Comment:
I believe the option name should be fixed at line 159 instead.

Reply:
Sure, I can make that change